etna/datasets/tsdataset.py

```diff
@@ -416,10 +417,26 @@ def inverse_transform(self, transforms: Sequence["Transform"]):
         Applied in reversed order.
         """
         # TODO: return regressors after inverse_transform
+        inverse_transform_target_components_flg = self.target_components is not None
+        target_df, target_components_df = None, None
+        if inverse_transform_target_components_flg:
+            target_df = self.to_pandas(features=["target"])
+            target_components_df = self.get_target_components()
+            self.drop_target_components()
```

Comment:
Why do we need to delete them here?

Reply:
To prevent inverse transform them in Transform each time, this way we transform them only once

Reply:
Can smth go wrong if we applied transform to the components? (it looks like a wrong scenario actually, but may be there is some valid one).

Reply:
Not sure that we expect this scenario, it is the same as transform quantile